plugins/providers/shield/client.go

```diff
@@ -145,13 +108,13 @@ func (c *client) GetAdminsOfGivenResourceType(ctx context.Context, id string, re
 	return userEmails, err
 }
 
-func (c *client) GetTeams(ctx context.Context) ([]*Team, error) {
+func (c *client) GetTeams(ctx context.Context) ([]*Group, error) {
```

Comment:
This is confusing, why is the function name GetTeams and it returns Group. Can we change the function name to GetGroups.

Reply:
ok